BLSQ/iaso
# POLIO-1789: migrate DG fields to ORPG fields

Copy value of ORPG date and quantities fields in corresponding DG fields

Related JIRA tickets : IA-XXX, WC2-XXX, POLIO-1789

## Self proofreading checklist


## Doc

Tell us where the doc can be found (docs folder, wiki, in the code...).

## Changes
- add migration to copy value of DG date and quantities fields in corresponding ORPG fields

## How to test

In supplychain: Find a VRF without value for date DG approval and Quantities approved by DG in doses

run the migration

Reload page: the fields should have the values of the corresponding ORPG fields

## Print screen / video

![Screenshot 2024-12-11 at 17 18 50](https://github.com/user-attachments/assets/50f0af41-4bc3-4194-948e-c4a34081e3b7)


## Follow the Conventional Commits specification

The **merge message** of a pull request must follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) specification.

This convention helps to automatically generate release notes.

Use lowercase for consistency.

[Example](https://github.com/BLSQ/iaso/commit/8b8d7d3064138c1e57878f17b4eb922516ab0112):

```
fix: empty instance pop up

Refs: IA-3665
```

Note that the Jira reference is preceded by a _line break_.

Both the line break and the Jira reference are entered in the _Add an optional extended description…_ field.
